File: GrandHotel-hotel reservation/WindowsFormsApp1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Windows.Forms.DataVisualization.Charting;

namespace WindowsFormsApp1
{
    public partial class Form1 : Form
    {
        public Modul mo =  new Modul();
        public static string username;
        public static int idpetugas;
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            mo.clearform(groupBox1);
        }

        private void button2_Click(object sender, EventArgs e)
        {
            string sql = $"SELECT *  FROM [GrandHotel].[dbo].[VE] WHERE Username='{textBox1.Text}' AND Password='{mo.hash256(textBox2.Text)}'";
            if (mo.getcount(sql)==0)
            {
                MessageBox.Show("Email atau Password salah");
            } else
            {
                username = mo.getvalue(sql, "Username");
                idpetugas = mo.getvalueint(sql, "ID");
                string role = mo.getvalue(sql, "Role");
                if (role == "Front Office")
                {
                    FrontOffice p = new FrontOffice();
                    p.Show();
                    this.Hide();

                }
                else if (role == "Housekeeper Supervisor")
                {
                    HousekeeperSupervisor p = new HousekeeperSupervisor();
                    p.Show();
                    this.Hide();

                }
                else
                {
                    Housekeeper p = new Housekeeper();
                    p.Show();
                    this.Hide();
                }
                //MessageBox.Show(role);
            }
        }
    }
}

File: Laundry App/laundryesemka/Form1.vb
Public Class Form1

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim sql = $"SELECT TOP (1000) [Id]
      ,[Name]
      ,[Password]
      ,[Email]
      ,[Address]
      ,[PhoneNumber]
      ,[IdJob]
      ,[DateOfBirth]
      ,[Salary]
  FROM [DBmeja_02].[dbo].[Employee] WHERE Email='{TextBox1.Text}' AND Password='{hash256(TextBox2.Text)}'"


        If adakosong(GroupBox1) Then
            MsgBox(kosong)
        Else
            If exc(sql) Then
                'MsgBox(sql)
                Dim role = getValue(sql, "idJob")
                username = getValue(sql, "Name")
                'MsgBox(username)
                'idlogin = getValue(sql, "Id")
                mainForm.Show()
                Me.Hide()
                'MsgBox(idlogin.GetType.ToString)
                'If role.Equals(1) Then
                '    mainForm.Show()
                '    Me.Hide()
                'Else
                '    laundressForm.Show()
                '    Me.Hide()

                'End If
            End If
        End If
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        clearForm(GroupBox1)
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        TextBox1.Select()

    End Sub
End Class
